src/psyclone/domain/lfric/kernel/inter_grid_vector_arg.py

```diff
@@ -76,13 +78,24 @@ def create_from_fparser2(fparser2_tree):
         :returns: an instance of InterGridVectorArg.
         :rtype: :py:class:`psyclone.domain.lfric.kernel.InterGridArg`
 
+        raises ValueError: if the metadata is not in the correct form.
```

Comment:
Missing leading colon.

Reply:
Actually, this method no longer raises an exception so I've removed the line (along with all the other classes that had this as well).